src/main/java/uk/gov/hmcts/reform/idam/client/models/UserDetails.java

```diff
@@ -3,22 +3,27 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 
 import java.util.List;
 import java.util.Optional;
 
 @Getter
 @Builder
+@EqualsAndHashCode
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class UserDetails {
 
-    private final String id;
-    private final String email;
+    private String id;
+    private String email;
+    private String forename;
+    private String surname;
+    private List<String> roles;
 
-    private final String forename;
-    private final String surname;
-    private final List<String> roles;
+    public UserDetails() {
+        super();
```

Comment:
unneeded?
```suggestion
```

Reply:
this is needed for WireMock stubs to work